### ISSUE TITLE
Should we worry about what happens to non-textual items within the text?

### ISSUE THREAD

**Issue by rralley** [UDT-ITF/website#52]
If a text contains images, or tables, or music, or anything else not covered by the spec, what should happen? Do we have to say, or is it just something that must be dealt with prior to supplying the text or text fragments in accordance with the API?

In case a concrete example is needed, the Newton Project has texts such as https://www.newtonproject.ox.ac.uk/view/texts/diplomatic/NATP00004 that include images, which appear on the site as img tags in the HTML. (Conversely, I've just looked up a seventeenth-century printed book on the Oxford Text Archive and found that in the HTML version it says '\[figure\]' where an image was in the original book, but I would surmise that that editorial remark is in the original transcription.)

**Comment by neilsjefferies** [comment]
The Fragment API does not deal with non-text items in any way. A manifest could include those details but use a IIIF reference or something else to retrieve them. Similarly, not all text in an object might be retrievable through a particular mode.

**Comment by rralley** [comment]
Makes sense to me. Do we need to set this out somewhere (e.g. in discussion the manifest, and/or in implementation notes)? Or do we treat it as obvious?

(I don't know about the other resources, but I don't think this will affect Casebooks because we don't actually reproduce the astrological charts in the entries. The only vague possibility is that there's a simple table somewhere that I've forgotten about.)

**Comment by mjhawkins** [comment]
Just to clarify. I get that the fragmenter doesn't deal with them - re: how you specify start/end points in the text. But, that doesn't necessarily prevent the web-deliverable version from having references to static images (or such like) at the appropriate points in the text, does it? The images/diagrams in Newton's maths and natural philosophical writings are central for readers understanding his text.
